gui/static/js/components/controlpanel.js

```diff
@@ -29,17 +29,25 @@ export default class ControlPanel extends React.Component {
 
     onKeyDown(event){
         if (this.state.keyboard){
-            if (event.key == 'w'){
-                this.sendMotors(100, 100, 100, 100);
+           if(event.keyCode==87) {
+                setTimeout(this.sendMotors(100,100,100,100),500);
```

Comment:
Just curious. Why is there a timeout function here? Did it not work when you executed it immediately? Wouldn't this mean that the motor stops before executing a forward/backward/whatever command, since `this.sendMotors(0,0,0,0)` follows right afterwards without a timeout function?

Reply:
I tried this on the robot, setTime is sent commends this.sendMotors(100, 100, 100, 100) continously for 0.5s,  and then it would stop .